Makefile

```diff
@@ -11,8 +11,8 @@ VERSION ?= $(shell cat hokusai/VERSION)
 MINOR_VERSION ?= $(shell cat hokusai/VERSION | awk -F"." '{ print $$1"."$$2 }')
 
 dependencies:
-	pip install --requirement requirements.txt --quiet
```

Comment:
pipenv should just make this part not necessary

Reply:
🎉